spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -311,26 +321,36 @@ function onRecvPacket(packet: Packet) {
        amount: packet.amount
      }
      tokens = []Token{token}
+     receiver = packet.receiver
   } else if transferVersion == "ics20-2" {
     FungibleTokenPacketDataV2 data = UnmarshalJSON(packet.data)
     tokens = data.tokens
+    // if we need to forward the tokens onward
+    // temporarily send to the channel escrow address of the intended receiver
+    if len(forwardingPath) > 0 {
```

Comment:
For consistency we should use `len(forwardingPath)>0` or `forwardingPath != nil` in both lines 330 and 398.